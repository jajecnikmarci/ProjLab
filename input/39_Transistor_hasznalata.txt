starttest
add room room1 3
add room room2 3
add room room3 3
add door room1 room2 door1
add door room2 room3 door2
add item room1 Transistor transistor1
add item room1 Transistor transistor2
add person room1 Student student1
pickup student1
pickup student1
use student1 transistor1
use student1 transistor1
gotoroom student1 room2
gotoroom student1 room3
printstate student1
printstate transistor1
printstate transistor2
use student1 transistor2
printstate student1
printstate room1
printstate room3
endtest