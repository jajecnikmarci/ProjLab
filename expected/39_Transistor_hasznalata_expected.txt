student1
	- location: room3
	- stunned: false
	- inventory: 
		transistor2
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
transistor1
	- pair: transistor2
	- room: room1
	- owner: student1
transistor2
	- pair: transistor1
	- room: 
	- owner: student1
student1
	- location: room1
	- stunned: false
	- inventory: 
		transistor1
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
room1
	- capacity: 3
	- people: 
		student1
	- doors: 
		door1
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
room3
	- capacity: 3
	- people: 
	- doors: 
		door2
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
